test/mpi/coll/test_coll_algos.sh

```diff
@@ -27,6 +27,7 @@ for algo_name in ${algo_names}; do
 
         coll_algo_tests+="bcasttest 10 ${env}${nl}"
         coll_algo_tests+="bcastzerotype 5 ${env}${nl}"
+        coll_algo_tests+="bcasttest 10 ${env} env=MPIR_CVAR_IBCAST_TREE_PIPELINE_CHUNK_SIZE=4096 ${nl}"
```

Comment:
Why the difference with how the chunk size envvar is added for reduce below?

Reply:
I can now make it same as reduce. Earlier there were some other bcast tests that would have taken a lot of time with pipelining, so I was selectively adding pipelining to tests. But those bcast tests are now gone.